express/src/middlewares/index.ts

```diff
@@ -0,0 +1,106 @@
+import { Request, Response, NextFunction, Errback } from "express";
+import morgan from "morgan";
+import chalk from "chalk";
+
+export const TransformResMiddlerware = (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  const oldSend = res.json;
+  res.json = (data) => {
+    // console.log(data); // do something with the data
+    const statusCode = data?.statusCode;
+    const message = data?.message;
+    const error = data?.error;
+    res.json = oldSend; // set function back to avoid the 'double-send'
```

Comment:
overwrite self?

Reply:
@czhen-zilliz express looks like need overwrite res.json to do response format

Reply:
Overwrite res.json/res.send to be consistent with nest.js response:
1. Format data in res like: `{ data: <raw data>, statusCode: 200 }`.
2. Format error data like: `{ error: <error details>, message: <error messages>, statusCode: 500 }`.